multichain-testing/tools/sleep.ts

```diff
@@ -1,7 +1,9 @@
 import type { LogFn } from 'ava';
 
-export const sleep = (ms: number, log?: LogFn) =>
+type NoOpFn = () => void;
+
+export const sleep = (ms: number, log: LogFn | NoOpFn = () => {}) =>
```

Comment:
Defining `NoOpFn` is awkward. The clean solution is to not make sleep() depend on Ava's special `.log` that has `.skip`. No reason for sleep() to know about a testing framework.

```suggestion
type Log = (...values: unknown[]) => void;

export const sleep = (ms: number, log: Log = () => {}) =>
```